react/src/components/CarouselTextbox.module.css

```diff
@@ -0,0 +1,259 @@
+    
+    /* UI Properties */
+    background-color: var(--color-header);
+}
+
+#textBoxHeader p {
+    /* Self Layout Properties */
+    margin: 0;
+
+    /* UI Properties */
+    text-align: left;
+    font: var(--font-primary);
+    font-style: italic;
+}
+
+#textBoxContent {
+    /* Self Layout Properties */
+    margin: 0;
+}
+
+.textBoxItemText {
+    /* Self Layout Properties */
+    margin: 0;
+
+    /* UI Properties */
+    text-align: left;
+}
+
+/* Smartphone & Tablet Layout */
+@media only screen and (max-width: 61.99rem) {
+    #textBoxHeader p {
+        /* Self Layout Properties */
+        margin: 0;
+    }
+
+    .textBoxItemText {
+        /* UI Properties */
+        font: var(--font-primary);
+        font-size: 0.9rem;
+    }
+}
+
+/* Tablet & Desktop Layout */
+@media only screen and (min-width: 37.51rem) {
+    #textBoxHeader p {
+        /* UI Properties */
+        font-size: 3rem;
+        letter-spacing: 0.9rem;
+    }
+
+    #textBoxContent {
+        /* Content Layout Properties */
+        display: flex;
+        flex-wrap: wrap;
+        align-content: space-evenly;
+
+        /* UI Properties */
+        list-style-type: none;
+        padding-left: 0;
+    }
+    
+    .textBoxItem {
+        /* Content Layout Properties */
+        display: flex;
+        align-items: flex-end;
+        justify-content: space-evenly;
+    }
+
+    .textBoxItemNumber {
+        /* Self Layout Properties */
+        height: 100%;
+    }
+    
+    .textBoxItemNumber p {
+        /* Self Layout Properties */
+        margin: 0;
+        padding-bottom: 0.5rem;
+        
+        /* UI Properites */
+        color: var(--color-darkGray);
+        font: var(--font-lato-bold);
+        font-size: 2.2rem;
+    }
+    
+    .textBoxItemNumber div {
+        /* Self Layout Properties */
+        padding-bottom: 0.5rem;
+        width: 100%;
+    
+        /* UI Properties */
+        background-color: var(--color-itemNumberDiv);
+    }
+}
+
+/* Smartphone Layout */
+@media only screen and (max-width: 37.5rem) {
+    #textBoxHeader {
+        /* Self Layout Properties */
+        padding: 0.88rem 1.3rem;
+        height: 17.55%;
+    }
+    
+    #textBoxHeader p {
+        /* Self Layout Properties */
+        padding-left: 4.6%;
+    
+        /* UI Properties */
+        font-size: 2.4rem;
+        letter-spacing: 0.7rem;
+    }
+
+    #textBoxContent {
+        /* Self Layout Properties */
+        padding: 0 0.6rem 0.6rem 0.6rem;
+
+        /* UI Properties */
+        list-style-position: inside;
+        text-align: left;
+    }
+    
+    .textBoxItem {
+        /* Self Layout Properties */
+        padding-top: 0.5rem;
+    }
+
+    .textBoxItemNumber {
+        /* Content Layout Properties */
+        display: none;
+    }
+
+    .textBoxItemText {
+        /* Content Layout Properties */
+        display: inline;
+    }
+}
+
+/* Tablet Layout */
+@media only screen and (min-width: 37.51rem) and (max-width: 61.99rem) {
```

Comment:
Generalnie, jak mówiłem o zamianie `px` na `rem` to zapomniałem o wspomnieć o tym, że nie stosuje się remów w media queries, tylko typowe wartości w pikselach, jak np. `1366px`, `1280px`, `768px` itd. Mea culpa

Reply:
Właśnie mi to nie pasowało, ale w internetach widziałem zaskakująco wiele przypadków takiego zastosowania `rem`ów.